.linters/cpp/checkKeyword.py

```diff
@@ -38,6 +38,7 @@
     'KW_WHEN',
     'KW_DELETE',
     'KW_FIND',
+    'KW_PATH',
```

Comment:
Add key work means incompatible. The user can't use `path` as label again.

Reply:
yes